src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -116,13 +116,16 @@ public function getConfigTreeBuilder()
                             ->addDefaultsIfNotSet()
                             ->children()
                                 ->booleanNode('enabled')->defaultTrue()->info('To enable or disable pagination for all resource collections by default.')->end()
+                                ->booleanNode('partial')->defaultFalse()->info('To enable or disable partial pagination for all resource collections by default when pagination is enabled.')->end()
                                 ->booleanNode('client_enabled')->defaultFalse()->info('To allow the client to enable or disable the pagination.')->end()
                                 ->booleanNode('client_items_per_page')->defaultFalse()->info('To allow the client to set the number of items per page.')->end()
+                                ->booleanNode('client_partial')->defaultFalse()->info('')->end()
```

Comment:
`info('')` must be filled (BTW, I'm not sure that it's really necessary to be able to set this option client-side, but as it's implemented, keep it)